spec/mandriller/base_spec.rb

```diff
@@ -19,16 +19,7 @@
   end
   subject { klass.foo }
 
-  BOOLEAN_SETTINGS = {
-    auto_text:                 'X-MC-Autotext',
-    auto_html:                 'X-MC-AutoHtml',
-    url_strip_qs:              'X-MC-URLStripQS',
-    preserve_recipients:       'X-MC-PreserveRecipients',
-    inline_css:                'X-MC-InlineCSS',
-    view_content_link:         'X-MC-ViewContentLink',
-    important:                 'X-MC-Important',
-  }
-  BOOLEAN_SETTINGS.each do |key, header|
+  described_class::BOOLEAN_SETTINGS.each do |key, header|
```

Comment:
I purposely did this because I want to avoid the unintentional change in this constant.